### PR TITLE
Adding to documentation for psi-samples

### PR DESCRIPTION
**Pull request by jennykim515**
I've tried a few of the samples on Linux and Windows and added additional documentation where I got stuck and faced some issues. I've had no prior experience in C# or psi before trying these, so please let me know where my comments are redundant or inaccurate!

### CODE REVIEW

File: Samples/HelloWorld/README.md [hunk 1]
```diff
@@ -8,6 +8,20 @@ Ensuring that this sample builds and runs correctly is a good way to verify that
 
 To build \\psi applications, we recommend using [Visual Studio 2019](https://www.visualstudio.com/vs/ "Visual Studio 2019") on Windows (the free, Community Edition is sufficient). Under Linux, we recommend using [Visual Studio Code](https://code.visualstudio.com/). We will build this sample application using the available [\\psi Nuget packages](https://github.com/microsoft/psi/wiki/Using-via-NuGet-Packages). 
 
+### Setting up Visual Studio 2019
+
+To later be able to use PsiStudio, install Visual Studio with the following features:
```
Comment (sandrist) [inline Samples/HelloWorld/README.md:13]: >To later be able to use PsiStudio [](http://example.com/codeflow?start=0&length=33)

We aren't persisting or visualizing any data with PsiStudio in this specific sample, so let's keep this section out for now

File: Samples/HelloWorld/README.md [hunk 2]
```diff
@@ -124,6 +140,21 @@ public static void Main()
 
 Try it out! Running this application should result in a "Hello World" message being printed to the console every second. Striking any key will close the application.
 
+## Running program on Linux
```
Comment (sandrist) [inline Samples/HelloWorld/README.md:143]: Thanks so much for adding these Linux-specific instructions!

File: Samples/RosTurtleSample/README.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ Since this is not a standard message type, we will need to define it ourselves i
 
 ## \psi World
 
-First, we'll create a console app and reference the `Microsoft.Psi` and `Microsoft.Ros` NuGet packages.
+First, we'll create a console app and reference the `Microsoft.Psi` and `Microsoft.Ros` NuGet packages. The ROS bridge from the Microsoft.Psi.ROS library allows creating ROS nodes using .NET possible, and it is only referenced within the ROS world (in Turtle.cs).
```
Comment (sandrist) [inline Samples/RosTurtleSample/README.md:70]: >possible [](http://example.com/codeflow?start=191&length=8)

remove "possible"

**Review by sandrist — CHANGES_REQUESTED** [review]
:clock1:

**Comment by sandrist** [comment]
Thanks for these additions to the documentation! Nice to see more of the Linux steps explicitly laid out. Just a couple small comments on minor edits to make, but otherwise this looks good to go.

**Review by AshleyF — APPROVED** [review]

**Review by sandrist — APPROVED** [review]
:shipit:

**Pull request merged**